realm/realm-library/src/main/java/io/realm/internal/TableView.java

```diff
@@ -510,8 +510,9 @@ public long upperBoundLong(long columnIndex, long value) {
     public TableView findAllLong(long columnIndex, long value){
         // Execute the disposal of abandoned realm objects each time a new realm object is created
         context.executeDelayedDisposal();
-        long nativeViewPtr = nativeFindAllInt(nativePtr, columnIndex, value);
-        try { 
+        long nativeViewPtr = 0;
+        try {
+            nativeViewPtr = nativeFindAllInt(nativePtr, columnIndex, value);
```

Comment:
@beeender A `nativeViewPtr` before `try-catch` block is just a placeholder, initialized to have a null-value for 1) steering clear of compiler warning and 2) providing a measure to check if native resource indeed is allocated even if `new TableView`fails.

 ( the `native*` series in this PR allocates native resource within them. we can further check with if `nativeViewPtr` is really null.)